### ISSUE TITLE
Default system timezone is Etc/UCT

### ISSUE THREAD

**Issue by dogilvie** [corretto/corretto-11#62]
### Describe the bug
The java system default timezone on an Amazon Linux 2 system shows as Etc/UCT.

Expected was any variant of UTC. This propagates to the user.timezone system property of the running jvm.

Google does not seem to know a lot about UCT as a timezone. Given it's rarity it seems to me to be dangerous to select this as the java default system timezone representation, even if it is equivalent to UTC.

### To Reproduce
On our Amazon Linux 2 system running Corretto 11
```
import java.time.ZoneId;
public class TZ {
public static void main(String[] args)
{
System.out.println(ZoneId.systemDefault());
}
}
```
$ java -cp . TZ
Etc/UCT

### Expected behavior
Expected to see Etc/UTC or UTC

### Platform information

OS: Amazon Linux 2

$ uname -a
Linux ip-10-93-1-82.us-west-2.compute.internal 4.14.146-120.181.amzn2.x86_64 #1 SMP Fri Oct 18 17:01:06 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

$ java -version
openjdk version "11.0.5" 2019-10-15 LTS
OpenJDK Runtime Environment Corretto-11.0.5.10.1 (build 11.0.5+10-LTS)
OpenJDK 64-Bit Server VM Corretto-11.0.5.10.1 (build 11.0.5+10-LTS, mixed mode)

$ cat /etc/sysconfig/clock
ZONE="UTC"
UTC=true
$ date
Wed Dec  4 02:03:24 UTC 2019

**Comment by dogilvie** [comment]
Possibly, this can be closed as I think this may be referred to here:

https://bugs.openjdk.java.net/browse/JDK-8223490

**Comment by alvdavi** [comment]
Hi,

I think you're right, as this seems to be JDK-8223490. I've tested and see that on AL2, `/etc/localtime` is not a symlink and that there is no `/etc/timezone`.

JDK-8223490 will be fixed on 11.0.6.

**Comment by davecurrie** [comment]
Fixed in 11.0.6.
